issues-guide.md

```diff
@@ -0,0 +1,100 @@
+Issues Guide
+=============
+This is a guide for collaborators and non-collaborators on how Github issues
+are handled and processed. 
+
+
+The Sprint
+-----------
+We are on a weekly sprint cycle and every Monday we get together for a meeting
+on Google Hangouts. Currently our meetings are at 12pm PDT / 7pm UTC and the 
```

Comment:
12:15 PDT / 19:15 UTC?  At least, that's what we have [here][1] since ipfs/pm@917cd424 (Update README.md, 2015-04-24).

[1]: https://github.com/ipfs/pm#sprints